lib/Subscriber/Core/RegistratorSubscriber.php

```diff
@@ -83,7 +84,7 @@ public function configureOptions(ConfigureOptionsEvent $event)
      *
      * @param HydrateEvent $event
      */
-    public function handleDefaultLocale(HydrateEvent $event)
+    public function handleDefaultLocale(AbstractMappingEvent $event)
```

Comment:
I still don't like the concept of a default locale when persisting, and Sulu already relies on not passing a locale in certain situations. I guess this change would make all the subscribers checking on the locale changing the behavior? Is this change necessary to these try/catches in the `MappingSubscriber`?